devops/ci/Dockerfile.integration.tmpl

```diff
@@ -0,0 +1,21 @@
+FROM {parent}
+
+USER root
+
+# docker
+COPY setup_docker.sh /tmp/
+RUN bash /tmp/setup_docker.sh \
```

Comment:
Can all the run statements be clubbed to call a utility script?
Would reduce number of intermittent images.

Reply:
It's not always a bad idea actually. If we combine docker setup with other pkgs installation then docker would be re-installed each minor change in not related stuff. Let's keep that discussion separately.